### ISSUE TITLE
Visualisation after normalise function

### ISSUE THREAD

**Issue by aghozlane** [mcanouil/NACHO#26]
Hi
I get a crash every time when I try to visualise the data after "renormalisation"  even with the example data:
```
Error in visualise(nacho_norm) : 
  [NACHO] Mandatory fields are missing in "nacho_norm"!
  "load_rcc()" and/or "normalise()" must be called before "visualise()".
```
Here the code that I copy paste, I have this issue R 3.6.1/3.6.2 linux and mac.
```r
  library(GEOquery)
  library(NACHO)

  # Import data from GEO
  gse <- GEOquery::getGEO(GEO = "GSE74821")
  targets <- Biobase::pData(Biobase::phenoData(gse[[1]]))
  GEOquery::getGEOSuppFiles(GEO = "GSE74821", baseDir = tempdir())
  utils::untar(
    tarfile = file.path(tempdir(), "GSE74821", "GSE74821_RAW.tar"),
    exdir = file.path(tempdir(), "GSE74821")
  )
  targets$IDFILE <- list.files(
    path = file.path(tempdir(), "GSE74821"),
    pattern = ".RCC.gz$"
  )
  targets[] <- lapply(X = targets, FUN = iconv, from = "latin1", to = "ASCII")
  utils::write.csv(
    x = targets,
    file = file.path(tempdir(), "GSE74821", "Samplesheet.csv")
  )

  # Read RCC files and format
  nacho <- load_rcc(
    data_directory = file.path(tempdir(), "GSE74821"),
    ssheet_csv = file.path(tempdir(), "GSE74821", "Samplesheet.csv"),
    id_colname = "IDFILE"
  )
  nacho_norm <- normalise(
    nacho_object = nacho,
    normalisation_method = "GLM",
    remove_outliers = TRUE
  )
  visualise(nacho_norm)
```

**Comment by mcanouil** [comment]
Hi,

Thanks for spotting this.
This is now fixed in 79448f2
``` r
# the development version from GitHub:
# install.packages("remotes")
remotes::install_github("mcanouil/NACHO")
```

**Comment by mcanouil** [comment]
The fix is part of NACHO v1.0.2 (#28) and it is now on CRAN.
